lib/install.ps1

```diff
@@ -34,7 +34,18 @@ function install_app($app, $architecture, $global, $suggested, $use_cache = $tru
         return
     }
 
-    write-output "Installing '$app' ($version) [$architecture]"
+    if (get_config 'manifest-review' $false) {
+        $show_manifest = read-host -Prompt "Show manifest? [y/N]"
```

Comment:
This statement block can be removed, because if the user has already set `manifest-review` in config, no need to ask again.